wordpress-mobile/WordPress-Android
# Issue/15983 make comment text selectable

Fixes #15983

This PR makes comment text selectable in Site Comment Details and in Reader Comment List.

Text in those place was set to non-selectable long time ago, because of [this]( https://code.google.com/p/android/issues/detail?id=30961) issue, which scrolled the long TextView after it gained focus. That issue was since fixed, and does not happen in android versions supported by us.

During my tests I discovered another issue when text is set to selectable - with a very long comment, selecting a word at the top of it, and scrolling the screen down causes a crash. The issue is present on devices with api <=27. I did not particularly like any workarounds, so I decided to disable selectable text on affected devices. Majority of our users will still benefit from this update.

To test:
- Using a device with api <=27 navigate to Site Comments and open a comment.
- Confirm that the text of the comment is not selectable.
- Using a device with api <=27 navigate to Reader Post Comments.
- Confirm that the text of the comment is not selectable.

--

- Using a device with api >=28 navigate to Site Comments and open a comment.
- Confirm that the text of the comment is selectable.
- Using a device with api >=28 navigate to Reader Post Comments.
- Confirm that the text of the comment is selectable.
- In Reader comment, find a very long comment, that spans below the visible area.
- Select a word at the top of the comment and scroll the screen down (so the selected word will move up out of view).
- Confirm that nothing is crashing.

## Regression Notes
1. Potential unintended areas of impact
- Crash described in "To test" happens on devices with API 28+

2. What I did to test those areas of impact (or what existing automated tests I relied on)
- Manual testing.

3. What automated tests I added (or what prevented me from doing so)
- Not testable.

PR submission checklist:
